core/domain/prod_validators_test.py

```diff
@@ -1409,6 +1409,22 @@ def test_model_with_invalid_commit_cmd_schmea(self):
         self.run_job_and_check_output(
             expected_output, sort=True, literal_eval=False)
 
+    def test_model_with_invalid_commit_message_length(self):
+
+        self.model_instance_0.commit_message = 'a' * (
+            constants.MAX_COMMIT_MESSAGE_LENGTH + 1)
+        self.model_instance_0.update_timestamps()
+        self.model_instance_0.put()
+        expected_output = [
+            (
+                u'[u\'failed validation check for commit message check '
+                'of ExplorationCommitLogEntryModel\', '
+                '[u\'Entity id exploration-0-1: Commit message larger than '
+                'accepted length\']]'
+            ), u'[u\'fully-validated ExplorationCommitLogEntryModel\', 3]']
+        self.run_job_and_check_output(
+            expected_output, sort=True, literal_eval=False)
+
```

Comment:
Add test like this but for the `ExplorationSnapshotMetadataModel`, thanks.

Reply:
I have shifted tests from `prod_validators_test` to `base_model_validators_test`. So this thing can't be done as per my latest approach.

Reply:
But you can add a tests that are bit different but test it from the job run point of view too, right? So keep the test in base_model_validators_test.py but also add tests to prod_validators_test.py that check whether the error is produced.

Reply:
Done